src/deviceConstants.js

```diff
@@ -0,0 +1 @@
+export const MAX_MOBILE_WIDTH = 480;
```

Comment:
Just add this to the Component at the top. A file just for one constant seems like a bit much

Reply:
It will be removed